packages/attestation-service/src/index.ts

```diff
@@ -3,26 +3,34 @@ import express from 'express'
 import rateLimit from 'express-rate-limit'
 import requestIdMiddleware from 'express-request-id'
 import * as PromClient from 'prom-client'
-import { initializeDB, initializeKit } from './db'
-import { getAccountAddress, getAttestationSignerAddress } from './env'
+import { initializeDB, initializeKit, verifyConfiguration } from './db'
+import { fetchEnv, fetchEnvOrDefault } from './env'
 import { rootLogger } from './logger'
 import { asyncHandler, createValidatedHandler, loggerMiddleware } from './request'
 import { AttestationRequestType, handleAttestationRequest } from './requestHandlers/attestation'
+import { handleAttestationDeliveryStatus } from './requestHandlers/delivery'
 import { handleLivenessRequest } from './requestHandlers/liveness'
 import { handleStatusRequest, StatusRequestType } from './requestHandlers/status'
 import { handleTestAttestationRequest } from './requestHandlers/test_attestation'
-import { initializeSmsProviders } from './sms'
+import { initializeSmsProviders, smsProvidersWithDeliveryStatus } from './sms'
+
+const deliveryStatusURLForProviderType = (t: string) =>
+  `${fetchEnv('EXTERNAL_CALLBACK_HOSTPORT')}/delivery_status_${t}`
 
 async function init() {
   await initializeDB()
   await initializeKit()
-  // TODO: Validate that the attestation signer has been authorized by the account
-  getAttestationSignerAddress()
-  getAccountAddress()
-  await initializeSmsProviders()
+
+  // Verify configuration.
+  const verifyConfig = fetchEnvOrDefault('VERIFY_CONFIG_ON_STARTUP', '1').toLowerCase()
+  if (verifyConfig === '1' || verifyConfig === 'y') {
```

Comment:
do we expect someone might put 'y' instead?  what if they put 'true'?

Reply:
or yes